std/zip.d

```diff
@@ -376,7 +385,9 @@ final class ZipArchive
      * Returns: array representing the entire archive.
      */
     void[] build()
-    {   uint i;
+    {
+        import std.algorithm : sort;
```

Comment:
it's better to be more selective as this will avoid the need to parse all `std.algorithm` packages.
-> `std.algorithm.sorting : sort`;